randovania/games/fusion/logic_database/Sector 1 SRX.txt

```diff
@@ -320,12 +320,8 @@ Extra - room_id: [9]
               Charge Beam or Missiles
           All of the following:
               Any of the following:
```

Comment:
you could probably just remove this any block

Reply:
fixes